concepts/staking.md

```diff
@@ -6,40 +6,26 @@ description: Details of Staking with Angle Protocol
+A vast portion of the governance tokens are distributed through staking contracts by the protocol to protocol and agToken users. This include Hedging Agents, SLPs, and agToken LPs on specific pools.
 
 While Hedging Agents automatically can accumulate governance tokens automatically as they stay in the protocol, Standard Liquidity Providers as well as stable holders have to stake their tokens (sanTokens for SLPs and agTokens for stable holders) in specific contracts to earn governance tokens. The protocol leaves them the opportunity to stake directly and seamlessly from the app after having minted.
 
-{% hint style="info" %}
-Staking contracts of the protocol may not be released immediately at launch of the protocol on mainnet.
-{% endhint %}
-
 ## 💐 Token Distribution
 
-There is one specific staking contract per sanToken and per agToken. The staking contracts for Hedging Agents are directly the contracts in which HAs' positions are stored.
-
-The fewer tokens are staked, the more interesting it is to participate in this staking contract because a similar amount of governance tokens are shared among a smaller amount of people.
+The distribution of governance tokens between the different staking contracts (called gauges) happens through weekly votes from veANGLE token holders. They assign specific weights to the different gauges, and the sum of their votes will dictate the next rewards distribution.
 
-The distribution of governance tokens to the different staking contracts is done by a contract controlled by governance. This contract stores for each staking contract a set of parameters, like a duration for the distribution (100 blocks, a day, a week, a month, a year, ...), and a number of tokens to distribute across this duration.
-
-400,000,000 ANGLE (40% of supply) are planned to be distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
+400,000,000 ANGLE (40% of the total supply) is being distributed through staking. The amount being distributed are divided by 1.5^(1/52) = 1.007827 every week, equivalent to dividing the distribution by 1.50 every year.
 
 ![ANGLE Distribution](../.gitbook/assets/Liquidity-Mining-Distributed-Supply-Over-Time.png)
 
 We built some simulations to evaluate how supply evolves over time, you can take a look at this [Google Sheet](https://docs.google.com/spreadsheets/d/1yraSUH_7D-VMnCUsIYWWdW1pxL7bDxN3o0M5japQmeY/edit#gid=0).
 
 {% hint style="info" %}
-ANGLE token owners are able to participate to Snapshot votes each week to decide how to allocate the liquidity mining supply across different pools. This is detailed [here](../governance/voting.md)
+The voting process deciding the rewards allocation takes place each week, and is detailed [here](../governance/voting.md).
```

Comment:
This page is no longer going to be up to date after the upgrade we can change it